src/Microsoft.VisualStudio.Editors/PropPages/AdvBuildSettingsPropPage.vb

```diff
@@ -192,12 +192,10 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
 
         Private Function IsExeProject() As Boolean
 
-            Dim obj As Object = Nothing
             Dim OutputType As VSLangProj.prjOutputType
 
             Try
-                GetCurrentProperty(VsProjPropId.VBPROJPROPID_OutputType, "OutputType", obj)
-                OutputType = CType(obj, VSLangProj.prjOutputType)
+                OutputType = OutputTypeProperty
```

Comment:
No, it does not. The reason is, `OutputTypeProperty` tries to get the outputtype from `ProjectProperties`, supported by Desktop projects, first before going through CPS.